q.cpp

```diff
@@ -0,0 +1,245 @@
+#include <stdio.h>
```

Comment:
Should we keep this file in this PR or remove it? (I can get checkout out an specific commit from repo and keep the PR clean).